.travis.yml

```diff
@@ -2,44 +2,432 @@ sudo: required
 dist: trusty
 language: java
 jdk:
-  - openjdk7
-before_install:
-  - cat /etc/hosts # optionally check the content *before*
-  - sudo hostname "$(hostname | cut -c1-63)"
-  - sed -n '/^127.0.0.1/!p' /etc/hosts | sed -e '1 i\127.0.0.1 localhost' -e "1 i\127.0.0.1 $(hostname | cut -c1-63)" | sudo tee /etc/hosts
-  - cat /etc/hosts # optionally check the content *after*
-  - cat /proc/cpuinfo | grep cores | wc -l
-  - free -h
-install:
-  - hibench=$(pwd)
-  - cd /opt/
-  - wget http://d3kbcqa49mib13.cloudfront.net/spark-1.6.0-bin-hadoop2.6.tgz
-  - tar -xzf spark-1.6.0-bin-hadoop2.6.tgz
-  - wget https://archive.apache.org/dist/hadoop/core/hadoop-2.6.5/hadoop-2.6.5.tar.gz
-  - tar -xzf hadoop-2.6.5.tar.gz
-  - cd ${hibench}
-  - cp ./travis/spark-env.sh /opt/spark-1.6.0-bin-hadoop2.6/conf/
-  - cp ./travis/core-site.xml /opt/hadoop-2.6.5/etc/hadoop/
-  - cp ./travis/hdfs-site.xml /opt/hadoop-2.6.5/etc/hadoop/
-  - cp ./travis/mapred-site.xml /opt/hadoop-2.6.5/etc/hadoop/
-  - cp ./travis/yarn-site.xml /opt/hadoop-2.6.5/etc/hadoop/
-  - cp ./travis/hibench.conf ./conf/
-  - cp ./travis/benchmarks.lst ./conf/
-before_script:
-  - "export JAVA_OPTS=-Xmx512m"
-cache:
-  directories:
-  - $HOME/.m2
-script:
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.4 -Dscala=2.11
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.3 -Dscala=2.11
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.2 -Dscala=2.11
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.1 -Dscala=2.11
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=2.0 -Dscala=2.11
-  - mvn clean package -q -Dmaven.javadoc.skip=true -Dspark=1.6 -Dscala=2.10
-  - sudo -E ./travis/configssh.sh
-  - sudo -E ./travis/restart_hadoop_spark.sh
-  - cp ./travis/hadoop.conf ./conf/
-  - cp ./travis/spark.conf ./conf/
-  - /opt/hadoop-2.6.5/bin/yarn node -list 2
-  - sudo -E ./bin/run_all.sh
+  - openjdk8
+jobs:
+  include:
+    - name: hibench-mvn
```

Comment:
Name it as `multi-version-build`?

Reply:
Because `build` is better than `mvn`, and declare it as `multi-version`

Reply:
@gczsjdy  Thanks , it's good idea.